packages/modules-sdk/package.json

```diff
@@ -27,7 +27,10 @@
     "@medusajs/orchestration": "^0.2.0",
     "@medusajs/types": "^1.10.2",
     "@medusajs/utils": "^1.9.6",
+    "@mikro-orm/postgresql": "5.7.12",
```

Comment:
thought: I don't think we need this deps?
```suggestion
```

Reply:
@riqwan ping 😄

Reply:
Ah, sorry just saw this. I didn't realize this was put in review, switching back to draft. This needs more work to make it work with yarn 3/pnpm and nextjs. 

Yes you're right, this isn't needed here, there is a missing dependency somewhere in the dependency chain. I added it here to test to see if that worked.

Reply:
Do we need any of these dependencies for a release?

Reply:
@olivermrbl Just checked it again, looks like the modules-sdk is the end of that dependency chain. This dep will be needed.
<img width="879" alt="Screenshot 2023-09-06 at 09 24 58" src="https://github.com/medusajs/medusa/assets/5105988/4ff72430-7f17-4a30-9139-6c4ec71de923">

Tested it with the latest snapshot, it works now. 🎉

Reply:
The bottom screenshot is from within the `utils` package, so I would think it wouldn't be needed in `modules-sdk`, but not an expert on deps, so I am fine to merge as is 👍

Reply:
damn 🤦🏻 removing it.